.github/workflows/ci.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         coverage: [ 'none' ]
         include:
           - php-version: '7.4'
-            coverage: xdebug
+            coverage: none
```

Comment:
We don't need this any more either.  We should add PHP versions 7.4 through 8.3 for unit tests as a separate PR.  Having this is effectively the same has changing to `php-version: [ '7.2', '7.3', '7.4' ]` above.

For this PR to be 'no overall change', there should still be 26 checks.  There are 27 as it stands.

Reply:
> We should add PHP versions 7.4 through 8.3 for unit tests as a separate PR. 

Yes, absolutely.

Reply:
Done.